Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/Ride.hs

```diff
@@ -88,26 +88,20 @@ type API =
                     :> "cancel"
                     :> ReqBody '[JSON] CancelRideReq
                     :> Post '[JSON] RideCancel.CancelRideResp
+                    :<|> TokenAuth
+                    :> Capture "rideId" (Id Ride.Ride)
+                    :> "odometerReading"
+                    :> QueryParam "start" Bool
```

Comment:
Why this param?

Reply:
it is used to identify whether we want to get start image or end image ?